File: Components/BlogML/Xml/BlogMLNode.vb
Imports System.Xml.Serialization

Namespace BlogML.Xml
 <Serializable>
 Public MustInherit Class BlogMLNode

  <XmlAttribute("id")>
  Public Property ID As String

  <XmlElement("title")>
  Public Property Title As String

  <XmlAttribute("date-created", DataType:="dateTime")>
  Public Property DateCreated As DateTime = Now

  <XmlAttribute("date-modified", DataType:="dateTime")>
  Public Property DateModified As DateTime = Now

  <XmlAttribute("approved")>
  Public Property Approved As Boolean = True

 End Class
End Namespace

File: Server/Core/BlogML/Xml/BlogMLNode.cs
using System;
using System.Xml.Serialization;

namespace DotNetNuke.Modules.Blog.Core.BlogML.Xml
{
  [Serializable]
  public abstract class BlogMLNode
  {

    [XmlAttribute("id")]
    public string ID { get; set; }

    [XmlElement("title")]
    public string Title { get; set; }

    [XmlAttribute("date-created", DataType = "dateTime")]
    public DateTime DateCreated { get; set; } = DateTime.Now;

    [XmlAttribute("date-modified", DataType = "dateTime")]
    public DateTime DateModified { get; set; } = DateTime.Now;

    [XmlAttribute("approved")]
    public bool Approved { get; set; } = true;

  }
}